src/css/index.css

```diff
@@ -115,16 +115,38 @@ input[type="radio"]:checked {
     z-index: 1001;
     position: absolute;
     pointer-events: none;
-    height: 100px;
+    height: auto;
     box-sizing: border-box;
     opacity: 0;
-    padding: 1.25rem;
+    padding: 0.5rem;
+    border-radius: 0.25rem;
+    border: 1px solid gray;
+
+    left: 10%;
+    top: 10%;
```

Comment:
Add a note point to the 10% in JS: If one is changed the other is changed. (I'd also like to use css-modules, or something like that, and it may be possible to bring these closer together.)